maestro-driver-ios/maestro-driver-iosUITests/Routes/Handlers/ClearTextHandler.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import FlyingFox
+import os
+import XCTest
+import Network
+
+@MainActor
+struct ClearTextHandler: HTTPHandler {
+    private let typingFrequency = 30
+
+    let logger = Logger(
+        subsystem: Bundle.main.bundleIdentifier!,
+        category: String(describing: Self.self)
+    )
+
+    func handleRequest(_ request: HTTPRequest) async throws -> HTTPResponse {
+        let deleteText = String(repeating: XCUIKeyboardKey.delete.rawValue, count: 40)
```

Comment:
Following the docs from [eraseText](https://maestro.mobile.dev/reference/text-input#erase-text), we should probably allow the amount of characters to be deleted from the request. WDYT?

Reply:
👍

Reply:
Implemented!